cmd/panfs.go

```diff
@@ -1610,3 +1772,26 @@ func (fs *PANFSObjects) GetRawData(ctx context.Context, volume, file string, fn
 	}
 	return fn(f, "fs", fs.fsUUID, file, st.Size(), st.ModTime(), st.IsDir())
 }
+
+func (fs *PANFSObjects) isPanasasConfigAgentObject(bucket, object string) bool {
+	if fs == nil {
+		return false
+	}
+
+	if bucket != minioMetaBucket {
+		return false
+	}
+
+	switch {
+	case strings.HasPrefix(object, "buckets/"):
+		return false
+	case strings.HasPrefix(object, "multipart/"):
```

Comment:
As I understand we do not need to store multipart and tmp objects into the blobdb